aports/main/postmarketos-mkinitfs/mkinitfs.sh

```
@@ -100,6 +100,23 @@ get_binaries()
 	lddtree -l $BINARIES | sort -u
 }
 
+# Collect non-binary files for osk-sdl and its dependencies
+get_osk_config()
+{
+	fontpath=$(awk '/^keyboard-font/{print $3}' /etc/osk.conf)
+	if [ ! -f $fontpath ]; then
+		echo "WARNING: Font specified in /etc/osk.conf does not exist!!!"
```

How about changing that to one `!` (also in the line below)?

---

You mean only have 1 line warning? What about making this condition fatal somehow, since it means that the keyboard will not work?

---

What I meant was: replace the three exclamation marks `!!!` with one `!`.

You are right, it makes sense to let it fail at this condition. To do that, I would replace "WARNING" with "ERROR", and write `exit 1` below the echo.